tests/neg/i2033.scala

```diff
@@ -3,7 +3,7 @@ import collection._
 object Test {
   def check(obj: AnyRef): Unit = {
     val bos = new ByteArrayOutputStream()
-    val out = new ObjectOutputStream(println) // error
+    val out = new ObjectOutputStream(println)
```

Comment:
I guess the overload is now resolved, but it's still a warning (`OutputStream is not @FunctionalInterface`), right?

Reply:
Yes, you get:

```
6 |    val out = new ObjectOutputStream(println)
  |                                     ^^^^^^^
  |method println is eta-expanded even though java.io.OutputStream does not have the @FunctionalInterface annotation.
```